types/constants.d.ts

```diff
@@ -94,6 +94,7 @@ interface PluginNames {
   WIDGET: 'widget';
   LOGIN_BUTTON: 'loginButton';
   CONNECT: 'connect';
+  COMPOSE: 'compose';
```

Comment:
Another place, where we should define each plugin 😞

Reply:
it's type definitions, hope that after make auto generates `.d.ts` files, it will throw away

Reply:
Yes